src/main/java/de/tum/in/www1/artemis/domain/Exercise.java

```diff
@@ -183,6 +183,12 @@ public abstract class Exercise extends BaseExercise implements LearningObject {
     @Transient
     private Long numberOfRatingsTransient;
 
+    /**
+     * It is defined here for transferring the value from client.
```

Comment:
Shouldn't this be Part of a DTO? Instead of adding an additional attribute here that is only used in one rare case?
```suggestion
     * Used for receiving the value from client.
```

Reply:
We have agreed with Stephan that this is ok in this case.